spec/presenters/navigation_presenter_spec.rb

```diff
@@ -11,6 +11,7 @@
         let(:user) { build(:user, personal_key: 'abcdef') }
 
         it 'has a link to reset personal key' do
+          create(:profile, :active, :verified, user: user)
```

Comment:
I think we can use the `:proofed` trait with the user factory to get the same effect:

```rb
let(:user) { build(:user, :proofed) }
```

Reply:
Good call, done!